thirdparty/mbedtls/library/aesni.h

```diff
@@ -38,6 +38,11 @@
 #define MBEDTLS_HAVE_X86_64
 #endif
 
+#if defined(__TINYC__)
```

Comment:
Seems like it would be better to handle this in the previous `#if`, rather than letting it be set then unsetting it.

I'd get rid of this block, and make the following change starting on line 37:
```c
    ! defined(MBEDTLS_HAVE_X86_64) && \
    ! defined(__TINYC__)
```